instructure/canvas-android
# [MBL-16917][Student] Grades screen LGO

Test plan: 
- Verify that the quantitative data is not shown on the grades screen when restrict quantitative data is turned on. Neither on the total grade and neither on the assignment grades.
- Verify that nothing changed on the grades screen when quantitative data is not restricted.

refs: MBL-16917
affects: Student
release note: none

## Checklist
